src/soundbase.h

```diff
@@ -103,7 +117,9 @@ class CSoundBase : public QThread
 
     virtual float GetInOutLatencyMs() { return 0.0f; } // "0.0" means no latency is available
 
-    virtual void OpenDriverSetup() {}
+    virtual void           OpenDriverSetup() {}
+    virtual EPaApiSettings GetExtraSettings() { return PaApiNoExtraSettings; }
```

Comment:
Why do this line and the next need to be here?  `src/soundbase.h` shouldn't have driver or underlying API-specifics in.